Cargo.toml

```diff
@@ -75,6 +75,10 @@ perf_counter = ["pfm"]
 # CI scripts run those tests with this feature.
 mock_test = []
 
+# This feature is only used for benchmarks.
+# It will expose some private functions for benchmarking.
+bench = []
```

Comment:
I am OK with renaming it to `test_private`.

The reason for not requiring the "bench" feature for some test cases is because the current mock tests don't require any of the items exposed from the "bench" feature.  But it will probably make things clearer to unconditionally require the "bench" or "test_private" feature when doing benchmarks.